### PR TITLE
fix IDE updater commands

### PR DESCRIPTION
**Pull request by AlbyIanna**
### Motivation
IDE updater commands are not working how expected. They work, but they are not giving any feedback to the user.

### Change description
Remove useless commands. Leave only the "Check for Arduino IDE updates" command. When launching this command, if there is an update available, it opens the update dialog. If there isn't any update available or some error occurs, it shows a notification message.

### Reviewer checklist

* [ ] PR addresses a single concern.
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
* [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by fstasi — APPROVED** [review]

**Comment by per1234** [comment]
@AlbyIanna if you still want me to check this out, please do a rebase to trigger a build. The build workflow is configured to only run on PRs submitted to the `main` branch, so it didn't run on the previous push. Now that the base has been changed to `main` it will run on the next update to the PR.

But I see Francesco already gave it the :+1:, so it is good to merge right away.

**Comment by AlbyIanna** [comment]
@per1234 ✅ done!

**Comment by AlbyIanna** [comment]
@per1234 I've just realised I'm not updating the channel configuration immediately when the user changes the channel preference. It's not done until the next IDE start-up. I'm going to fix it in this PR because I think it's pertinent.

**Comment by AlbyIanna** [comment]
@per1234 Done. Now it should work as expected, that is when you change the channel or the update URL and then check for updates, the updater will use the correct channel/URL.

P.S.: I've noticed the expand/collapse icons in the bottom-right notifications are "invisible". Fixing it in another PR.

**Review by per1234 — CHANGES_REQUESTED** [review]

File: arduino-ide-extension/src/browser/ide-updater/ide-updater-commands.ts [hunk 1]
```diff
@@ -6,45 +6,41 @@ import {
...
-
-  quitAndInstall(): void {
-    this.updater.quitAndInstall();
+    try {
+      const updateInfo = await this.updater.checkForUpdates(initialCheck);
+      if (!!updateInfo) {
+        this.updaterDialog.open(updateInfo);
+      } else {
+        this.messageService.info(
+          `There are no recent updates available the Arduino IDE`
```
[the remark this answers is not in the document]
Comment (AlbyIanna) [reply arduino-ide-extension/src/browser/ide-updater/ide-updater-commands.ts:35]: ❤️

**Pull request merged**